### PR TITLE
Add some OpenSSL compat error strings.

### PR DESCRIPTION
**Pull request by anhu**
Fixes ZD 16534

### CODE REVIEW

**Comment by anhu** [comment]
jenkins retest this please

File: src/internal.c [hunk 1]
```diff
@@ -24411,10 +24409,37 @@ const char* wolfSSL_ERR_reason_error_string(unsigned long e)
     case HTTP_APPSTR_ERR:
         return "HTTP Application string error";
 #endif
-#ifdef OPENSSL_EXTRA
+#if defined(OPENSSL_EXTRA) || defined(OPENSSL_EXTRA_X509_SMALL) || \
```
Comment (JacobBarthelmeh) [inline src/internal.c:24412]: The OPENSSL_EXTRA_X509_SMALL initial work was trimmed where every byte of code size mattered. Can this section of extra error messages not be included for OPENSSL_EXTRA_X509_SMALL?

**Review by JacobBarthelmeh — CHANGES_REQUESTED** [review]

**Comment by anhu** [comment]
jenkins retest this please

**Comment by anhu** [comment]
jenkins retest this please

**Review by JacobBarthelmeh — APPROVED** [review]

**Pull request merged**